api/router.go

```diff
@@ -253,5 +261,13 @@ func NewRouter(s *Server) *mux.Router {
 	r.Path("/v1/ws/events").Handler(f(schemas, eventListStream))
 	r.Path("/v1/ws/{period}/events").Handler(f(schemas, eventListStream))
 
+	objectStoreListStream := NewStreamHandlerFunc("objectstores", s.wsc.NewWatcher("objectstore"), s.objectStoreList)
```

Comment:
Can we use objectStore instead of objectstore to be uniformed with other resources' naming?

```suggestion
	objectStoreListStream := NewStreamHandlerFunc("objectstores", s.wsc.NewWatcher("objectStore"), s.objectStoreList)
```

Reply:
ping @m-ildefons